### ISSUE TITLE
when i visit my profile i can't see the number of the SOLD NFTs till i click on it

### ISSUE THREAD

**Issue by onerawartist** [kodadot/nft-gallery#1230]
### What happened?

can't see the number of the sold nfts till i click on it
and i think its a must , because it helps give a good first impression of the artist ^^

### Please reproduce in steps

just visit my profile, or any profile

### Expected Behavior

to see the number of the SOLD nfts before clicking on it 

### What browsers are you seeing the problem on?

Chrome

### Are you logged in?

Yes

### At which address did you encounter bug?

nft.kodadot.xyz / kodadot.xyz

### Screenshots

Paste from your clipboard or upload screenshots
![01](https://user-images.githubusercontent.com/71472224/142730749-510f24b7-b262-4657-aac5-bfb349890260.png)
![02](https://user-images.githubusercontent.com/71472224/142730756-0b6d8713-7cd3-47fe-b033-4053db9c92c9.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by roiLeo** [comment]
ref #1133 & #1120 related

**Comment by onerawartist** [comment]
> ref #1133 & #1120 related

Those are when the creator has created nothing so its better to not show zeros 
 , but in this case i did creat and i did sell but my profile not showing it 
Is this a feature not a bug scenarios? lol

**Comment by yangwao** [comment]
> > ref #1133 & #1120 related
> 
> Those are when the creator has created nothing so its better to not show zeros , but in this case i did creat and i did sell but my profile not showing it Is this a feature not a bug scenarios? lol

Yeah, this is not queried until you click to save some queries as how RMRK works is quite heavy on querying and to speed up loading profiles, we don't do it right now. 

I got what impress you got when we don't show anything rn now. We will look on that, I've put it into backlog w/ p5

**Comment by yangwao** [comment]
Curious @onerawartist any got better now?

**Comment by onerawartist** [comment]
@yangwao everything has became much better, but this exact issue still the same

![Screenshot_2022-05-08-13-31-15-667_com.android.chrome.jpg](https://user-images.githubusercontent.com/71472224/167296266-40bc4c0b-d0c2-4c6d-86ce-d10e184a0853.jpg)

**Comment by roiLeo** [comment]
Closed in #3070 
![Capture d’écran 2022-06-07 à 3 37 50 PM](https://user-images.githubusercontent.com/9987732/172394333-93ed985c-233c-433f-a606-4d9aef2cd6f5.png)
